opentelemetry-sdk/benches/metric_counter.rs

```diff
@@ -50,12 +72,11 @@ fn counter_add(c: &mut Criterion) {
 
     c.bench_function("Counter_Add_Unsorted", |b| {
         b.iter(|| {
-            let mut rng = SmallRng::from_entropy();
             // 4*4*10*10 = 1600 time series.
-            let index_first_attribute = rng.gen_range(0..4);
-            let index_second_attribute = rng.gen_range(0..4);
-            let index_third_attribute = rng.gen_range(0..10);
-            let index_forth_attribute = rng.gen_range(0..10);
+            let index_first_attribute = CURRENT_RNG.with(|rng| rng.borrow_mut().gen_range(0..4));
+            let index_second_attribute = CURRENT_RNG.with(|rng| rng.borrow_mut().gen_range(0..4));
+            let index_third_attribute = CURRENT_RNG.with(|rng| rng.borrow_mut().gen_range(0..10));
+            let index_forth_attribute = CURRENT_RNG.with(|rng| rng.borrow_mut().gen_range(0..10));
```

Comment:
how about borrowing once, and generate all numbers in single closure:
```rust
    CURRENT_RNG.with(|rng| {
        let mut rng = rng.borrow_mut();
        let index_first_attribute = rng.gen_range(0..4);
        let index_second_attribute = rng.gen_range(0..4);
        let index_third_attribute = rng.gen_range(0..10);
        let index_forth_attribute = rng.gen_range(0..10);
        counter.add(...)
    });

```

Reply:
thanks! It boosted throughput further!